backend/src/game/player.py

```diff
@@ -0,0 +1,29 @@
+from backend.src.game import constants
+from backend.src.game.locations import Locations
+
+
+class Player:
+    def __init__(self):
+        self.maximum_health = constants.DEFAULT_HEALTH
+        self.current_health = self.maximum_health
+        self.location = Locations.OUTSIDE
+        self.is_alive = bool(1)
+        self.victory_points = constants.ZERO
+
+    def move_to_tokyo(self):
+        self.location = Locations.TOKYO
+
+    def leave_tokyo(self):
+        self.location = Locations.OUTSIDE
+
+    def change_health(self, change_integer):
```

Comment:
Due to the way the change health function and alive instance variable work, there is a known odd behavior here in that a player can be not alive but functions will still update the instance values.  Which means you could get things like a player could end up dying but if they were queued to do an action to add victory points, this would take place and the player could win the game.  There are side-effects here and there is definitely an argument that the game engine should handle all these circumstances not the player; however, it could make sense for a custom exception to be built that fires if an action is trying to be taken by a non-living player

Reply:
Card abilities will eventually mess with a lot of this stuff, so I left the ability there. The game should be responsible for not declaring a dead player a winner :P

Reply:
For this section of code, wondering why
        self.maximum_health = constants.DEFAULT_HEALTH
        self.current_health = self.maximum_health
instead of simply  
        self.maximum_health = constants.DEFAULT_HEALTH
        self.current_health = constants.DEFAULT_HEALTH
the current solution although the same as my suggestion, was that done for readability or some other game logic

Reply:
This also can just be reduced further
`
self.maximum_health = self.current_health = constants.DEFAULT_HEALTH
`

Reply:
Thank you!